demos/k8ssandra/k8ssandra.yaml

```diff
@@ -37326,6 +37349,9 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: px-k8ssandra/cass-operator-serving-cert
     controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
```

Comment:
What's the purpose of ensuring this is set? It seems like it was added for the `MutatingWebhookConfiguration` as well.

Reply:
That actually got set when I ran the `kustomize`... Maybe it happened because of differing kustomize versions since generating the YAMLs for this required a newer version of kustomize.

I'll add a separate PR for upgrading kustomize, just so that's consistent.